src/wp-includes/style-engine/class-wp-style-engine-css-rule.php

```diff
@@ -89,6 +100,28 @@ public function add_declarations( $declarations ) {
 		return $this;
 	}
 
+	/**
+	 * Sets the rules group.
+	 *
+	 * @param string $rules_group A parent CSS selector in the case of nested CSS, or a CSS nested @rule,
+	 *                            such as `@media (min-width: 80rem)` or `@layer module`.
+	 *
+	 * @return WP_Style_Engine_CSS_Rule Returns the object to allow chaining of methods.
+	 */
+	public function set_rules_group( $rules_group ) {
+		$this->rules_group = $rules_group;
+		return $this;
+	}
+
+	/**
+	 * Gets the rules group.
+	 *
```

Comment:
Should new methods and updates to other public methods/APIs get a `@since` annotation?

Reply:
Oh good point, yeah they should!

Reply:
Updated ✅